src/core/src/main/java/org/locationtech/geogig/api/hooks/Scripting.java

```diff
@@ -83,11 +83,8 @@ public static void runJVMScript(AbstractGeoGigOp<?> operation, File scriptFile)
             setParamMap((Map<String, Object>) map, operation);
         } catch (ScriptException e) {
             Throwable cause = Throwables.getRootCause(e);
-            // TODO: improve this hack to check exception type
             if (cause != e) {
                 String msg = cause.getMessage();
-                msg = msg.substring(CannotRunGeogigOperationException.class.getName().length() + 2,
```

Comment:
I seem to recall this is needed when running with a Java6 JVM. Java6 and Java7 come with different javascript engines, and the older one's exception handling is kind of tricky. @volaya may know better though, but for the time being I wouldn't remove it.